test/src/examples_complete_test.go

```diff
@@ -1,124 +0,0 @@
-package test
-
-import (
-  "os"
-  "strings"
-  "testing"
-
-  "github.com/gruntwork-io/terratest/modules/random"
-  "github.com/gruntwork-io/terratest/modules/terraform"
-  testStructure "github.com/gruntwork-io/terratest/modules/test-structure"
-  "github.com/stretchr/testify/assert"
-)
-
-func cleanup(t *testing.T, terraformOptions *terraform.Options, tempTestFolder string) {
-  terraform.Destroy(t, terraformOptions)
-  os.RemoveAll(tempTestFolder)
-}
-
-// Test the Terraform module in examples/complete using Terratest.
-func TestExamplesComplete(t *testing.T) {
```

Comment:
We will need to think about running tests around open source modules at some point. If we're serious we'll need it. And we want to be more serious for sure.